valve/addons/amxmodx/data/lang/agmodxllhl.txt

```diff
@@ -0,0 +1,21 @@
+[en]
+LLHL_FPSL_KICK = Your 'fps_max' value is Higher than %d
+LLHL_FPSL_KICK_MSG = %s has been kicked for exceeding the 'fps_max' limit
+LLHL_DEMO_RECORDING = Recording %s.dem
+LLHL_UNSTUCK_ON_COOLDOWN = You have to wait %.1f seconds to use the command again
+LLHL_UNSTUCK_FREESPOT_NOTFOUND = No free spot was found where you can relocate
+LLHL_UNSTUCK_PLAYER_DEAD = You can't use this command while you're dead
+LLHL_FILECONSISTENCY_MSG = %s (%s) has been kicked for having a different file than the one on the server: sound/%s
+LLHL_FILECONSISTENCY_KICK = You have a different file than the one on the server: %s
+LLHL_BLOCK_NAMECHANGE_MSG = You can't change your name while a match is in progress.
+
+[es]
+LLHL_FPSL_KICK = Tu valor de tu 'fps_max' es mayor que %d
+LLHL_FPSL_KICK_MSG = %s ha sido kickeado por superar el limite de 'fps_max'
+LLHL_DEMO_RECORDING = Grabando %s.dem
+LLHL_UNSTUCK_ON_COOLDOWN = Debes de esperar %.1f segundos para poder usar nuevamente el comando
+LLHL_UNSTUCK_FREESPOT_NOTFOUND = No se encontro ningun lugar libre en donde puedas reubicarte
```

Comment:
Maybe this suits better "No se encontro ningun lugar libre para poder destrabarte." or "No se encontro ningun lugar libre para reubicarte.".

Reply:
Done